utils/storager.go

```diff
@@ -0,0 +1,58 @@
+package utils
+
+import (
+	"fmt"
+	"sync/atomic"
+
+	_ "github.com/beyondstorage/go-service-memory"
+	"github.com/beyondstorage/go-storage/v4/services"
+	"github.com/beyondstorage/go-storage/v4/types"
+	"github.com/beyondstorage/go-stream"
+)
+
+const (
+	upperStorageConnString = "memory://"
+)
+
+var (
+	streamMap    map[string]*stream.Stream
```

Comment:
`stream` is designed for use concurrently, so we don't need to maintain multiple streams. Init one stream while the ftp server started is enough.